documents/pt_gsd-ud-train-0171.conllu

```diff
@@ -160,17 +160,16 @@
 43	o	o	DET	DET	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	44	det	_	_
 44	Sul	_	PROPN	PNOUN	_	40	nmod	_	SpaceAfter=No
 45	,	,	PUNCT	.	_	40	punct	_	_
+46-48	comparando-a	_	_	_	_	_	_	_	_
 46	comparando	comparar	VERB	VERB	_	26	acl	_	_
-47	-	-	PUNCT	.	_	46	punct	_	_
-48-49	à	_	_	_	_	_	_	_	_
```

Comment:
This one looks possibly bugged to begin with.  Not sure what it means to have the `comparando` followed by an MWT `à`.  Perhaps it shouldn't have been merged, but the `-` was pretty indicative of merging in most cases